firmware/hw_layer/drivers/sent/sent_hw_icu.cpp

```diff
@@ -50,29 +50,34 @@ static ICUConfig icucfg_in1 =
 	.arr = 0xFFFFFFFFU,
 };
 
+/* current config */
+static brain_input_pin_e sentPins[SENT_INPUT_COUNT];
```

Comment:
This falls outside of the pattern. It looks like the pattern is "activeConfiguration"

```
#if EFI_ALTERNATOR_CONTROL
	onConfigurationChangeAlternatorCallback(&activeConfiguration);
#endif /* EFI_ALTERNATOR_CONTROL */

#if EFI_BOOST_CONTROL
	onConfigurationChangeBoostCallback(&activeConfiguration);
#endif
#if EFI_ELECTRONIC_THROTTLE_BODY
	onConfigurationChangeElectronicThrottleCallback(&activeConfiguration);
#endif /* EFI_ELECTRONIC_THROTTLE_BODY */

#if EFI_ENGINE_CONTROL && EFI_PROD_CODE
	onConfigurationChangeBenchTest();
#endif

```